BTSPocket/Controllers/Profile/ProfileViewController.swift

```diff
@@ -21,18 +25,64 @@ class ProfileViewController: UIViewController {
     @IBOutlet weak var imageProfile: UIImageView!
```

Comment:
If you are going to use complete name of control as prefix like label, button, table, etc. So this outlet should be named as `imageviewProfile` avoiding any misunderstanding with **UIImage** object